conf/services.conf

```diff
@@ -0,0 +1,163 @@
+# Use Simple API to create endpoints for publicly accessible research frontend.
+simple_api_endpoints = {
+    # Library
+    library_search = {
+        type = search,
+        table = ca_objects,
+        restrictToTypes = [ library ],
+        checkAccess = [ 1 ],
+        content = {
+            object_id = ^ca_objects.object_id,
+            idno = ^ca_objects.idno,
+            Title = ^ca_objects.preferred_labels,
+            Author = ^ca_objects.Author,
+            Publisher = ^ca_objects.Publisher,
+            DateOfPublication = ^ca_objects.DateOfPublication,
+            PlaceOfPublication = ^ca_objects.PlaceOfPublication,
+            PublicationType = ^ca_objects.PublicationType,
+            Subjects = ^ca_occurrences
```

Comment:
According to the spec these should also be searchable (I see you've got them in the detail view):
```
            Series = ^ca_objects.Series,
            LibraryNumber = ^ca_objects.LibraryNumber,
            Donated = ^ca_objects.Donated
```

I'm also unclear as to whether Purchased, OrderNumber and Review should be added, but those can be added later.